enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -0,0 +1,10 @@
+<p>{% if user_name %}Dear {{ user_name }},{% else %}Hi!{% endif %}</p>
+<p>
+You have been enrolled in <a href="{{ course_url}}">{{ course_name }}</a>, a {% if program_type %}{{ program_type }} {% endif %}{{ course_type }} offered by {{ organization_name }}. This {{ course_type }} begins {{ start_date | date:"F j, Y" }}. For more information, see <a href="{{ course_url}}">{{ course_name }}</a>.
+</p>
+<p>
+Thanks,
+</p>
+<p>
+The {{course_name}} team
+</p>
```

Comment:
Re: the content: that's prescribed by the ticket, so I'm not sure if we want a different template, especially as we're using a different set of data here.

But I agree that there should be a preview view.